Changelog.md

```diff
@@ -9,6 +9,8 @@ Compiler Features:
  * Yul Optimizer: Add rule to convert `mod(mul(X, Y), A)` into `mulmod(X, Y, A)`, if `A` is a power of two.
  * Yul Optimizer: Add rule to convert `mod(add(X, Y), A)` into `addmod(X, Y, A)`, if `A` is a power of two.
  * Code Generator: More efficient code for checked addition and subtraction.
+ * Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the deployed bytecode.
```

Comment:
Why was all this renamed to include cbor? :)

Reply:
Because ``metadata`` so far always refers to the json metadata file and we don't want to overload nomenclature any further. But we just decided not to rush this into the release after all and take until the end of this week to decide the final naming.

Reply:
Well... "always" with the exception of standard-json's ``settings.metadata.bytecodeHash`` which kind of refers to both.

Reply:
@axic Are you ok with the name or do you think we should use "auxdata" or something else?